layouts/partials/general_header.html

```diff
@@ -0,0 +1,3 @@
+<div class="general-page-bg bg-center bg-cover relative flex items-center text-center justify-center z-10 h-44 lg:h-96 mt-0">
```

Comment:
this is some minor refactoring as I'm using the same header as the General Pages